index.html

```diff
@@ -0,0 +1,464 @@
+                multiple windows the user agent may use `new-client` to avoid data loss.
+              </p>
+            </dd>
+            <dt>
+              <dfn data-dfn-for="route to">new-client</dfn>
+            </dt>
+            <dd>
+              A new web app client is created to load the launch's target URL.
+            </dd>
+            <dt>
+              <dfn data-dfn-for="route to">existing-client-navigate</dfn>
+            </dt>
+            <dd>
+              If an existing web app client is open it is brought to focus and
+              navigated to the launch's target URL. If there are no existing web
+              app clients the [=route to/new-client=] behaviour is used instead.
+            </dd>
+            <dt>
+              <dfn data-dfn-for="route to">existing-client-retain</dfn>
+            </dt>
+            <dd>
+              If an existing web app client is open it is brought to focus but
+              not navigated to the launch's target URL, instead the target URL
+              is communicated via {{LaunchParams}} . If there are no existing
+              web app clients the [=route to/new-client=] behaviour is used
+              instead.
+            </dd>
+          </dl>
+        </p>
+        <p>
+          To <dfn>process the `route_to` member</dfn>, given [=ordered
+          map=] |json_launch_handler:ordered map|, [=ordered map=]
+          |manifest_launch_handler:ordered map|, run the following:
+        </p>
+        <ol class="algorithm">
+          <li>If |json_launch_handler|["route_to"] does not [=map/exist=],
+          return.
+          </li>
+          <li>If the type of |json_launch_handler|["route_to"] is
+          [=list=]:
+            <ol class="algorithm">
+              <li>[=list/For each=] |entry| of |json_launch_handler|["route_to"]:
+                <ol class="algorithm">
+                  <li>If the type of |entry| is not [=string=], continue.
+                  </li>
+                  <li>If |json_launch_handler|["route_to"] is supported by the user agent, set manifest_launch_handler["route_to"] to |entry| and return.
+                  </li>
+                </ol>
+              </li>
+            </ol>
+          </li>
+          <li>If the type of |json_launch_handler|["route_to"] is not [=string=], return.
+          </li>
+          <li>If |json_launch_handler|["route_to"] is supported by the user agent, set manifest_launch_handler["route_to"] to |json_launch_handler|["route_to"].
+          </li>
+        </ol>
+        <p class="note">
+          `route_to` accepts a list of strings as a future proofing mechanism. If a new [=route to target=] is added in the future but has not yet been implemented by all user agents, web developers can cater to the old user agents by providing fallback [=route to target=] values in a list format. User agents will use the first value that is valid to them.
```

Comment:
I wouldn't even explain this as a future-proofing mechanism, because it's also a present-proofing mechanism.

Since you've said: "User agents MAY support only a subset of the route to targets depending on the constraints of the platform", that means the list of strings is basically to let sites specify their behaviour preferences, then let the user agent choose the most preferred one that it supports.

Given that UAs do not need to support all of them, there is no need for a temporal / versioning explanation.

Reply:
Rewrote to be based on UA/platform support fallbacks.